service/controller/clusterapi/v29/resource/tcdp/create.go

```diff
@@ -63,16 +62,16 @@ func (r *Resource) EnsureCreated(ctx context.Context, obj interface{}) error {
 
 		// TODO
 		var params *template.ParamsMain
-		{
-			iamRoles, err := r.newIAMRolesParams(ctx, cr)
-			if err != nil {
-				return microerror.Mask(err)
-			}
-
-			params = &template.ParamsMain{
-				IAMRoles: iamRoles,
-			}
-		}
+		//		{
+		//			iamRoles, err := r.newIAMRolesParams(ctx, cr)
+		//			if err != nil {
+		//				return microerror.Mask(err)
+		//			}
+		//
+		//			params = &template.ParamsMain{
+		//				IAMRoles: iamRoles,
+		//			}
+		//		}
```

Comment:
This code will be dropped in the next PRs anyway when the real templates have to be made proper use of.

Reply:
I don't quite understand why we can't drop it now? Is it just to help remember for the actual implementation?

Reply:
This will get merge conflicts and I can just drop the commented lines. It is easier to scan for my eyes. I do not use these "smart" merge conflict resolvers that my Atom supports because it often does not work and is fucky sometimes with the formatting.

Reply:
You can get rid of the smart resolver AFAIK and git will insert the usual conflict lines into your files when a conflict has to be resolved.

Not a blocker for me here though.